operator-wg/whitepaper/02_Foundation.md

```diff
@@ -33,53 +32,64 @@ A general diagram of an operator will have software that can read the desired sp
-* Our domain specific knowledge that describes the state of the application in a declarative way.
+* Our domain-specific knowledge that describes the state of the application in a declarative way.
 * A controller that run continuously:
   * Reads and is aware of the state.
   * Runs actions against the application in an automated way.
-  * Report the state of the application  in a declarative way.
+  * Reports the state of the application  in a declarative way.
 
-This design pattern will be applied on Kubernetes and its operators in the next sections.
+This design pattern will be applied to Kubernetes and its Operators in the next sections.
 
 ### Operator Characteristics
 **(Current) Issue: https://github.com/cncf/sig-app-delivery/issues/37**
 
-The core purpose of any operator is to extend its orchestrator's underlying API with new domain knowledge. As an example, an orchestration platform as Kubernetes natively understands things like containers and layer 4 load balancers via the Pod and Service objects. An operator adds new capabilities for more complex systems and applications. For instance, a prometheus-operator introduces new object types, "Prometheus", which extend Kubernetes with high-level support for deploying and running Prometheus servers.
+The core purpose of any operator is to extend its framework's underlying API with new domain knowledge. As an example, an orchestration platform as Kubernetes natively understands things like containers and layer 4 load balancers via the Pod and Service objects. An operator adds new capabilities for more complex systems and applications. For instance, a Prometheus operator introduces new object types, "Prometheus", which extend Kubernetes with high-level support for deploying and running Prometheus servers.
 
 The capabilities provided by an operator can be sorted into three overarching categories: dynamic configuration, operational automation, and domain knowledge.
 
 #### Dynamic Configuration
-Since the dawn of time, there have been two main ways to configure software, configuration files and environment variables. In the cloud native world there are some newer options like querying a well-known API at startup, but most existing software in the world uses one or both of these options. Kubernetes naturally provides many tools to interact with these (ConfigMaps, Secrets, etc) but because they are generic, they don’t understand any specifics of configuring a given application. An operator can define new custom object types (custom resources) to better express the configuration of a particular application in a Kubernetes context.
+Since the dawn of time, there have been two main ways to configure software: configuration files and environment variables. In the cloud-native world there are some newer options like querying a well-known API at startup, but most existing software in the world uses one or both of these options.
```

Comment:
```suggestion
Since the early origins of modern computing, there have been two main ways to configure software; configuration files and environment variables. In the cloud-native world, there are some newer options like querying a well-known API at startup, but most existing software in the world uses one or both of these options.
```

It was a bit cheesy